lib/moxinet/response.ex

```diff
@@ -5,8 +5,9 @@ defmodule Moxinet.Response do
 
   @type t :: %__MODULE__{
           status: 100..600,
+          headers: %{binary() => binary()} | [{binary(), binary()}],
```

Comment:
```suggestion
          headers: [{binary(), binary()}],
```
Can't we just stick to how Plug.Conn represents headers?
https://hexdocs.pm/plug/Plug.Conn.html#t:headers/0